src/algebra/big_operators/nat_antidiagonal.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson
+-/
+
+import data.finset.nat_antidiagonal
+import algebra.big_operators
```

Comment:
Can this be minimized? Also, should this file be added to `algebra/big_operators/default.lean`?

Reply:
How do you mean minimized? `finset.sum_insert` lives in `big_operators`, and `finset.nat.antidiagonal_succ` lives in `finset.nat_antidiagonal`.

Reply:
Also, I think it may be useful to add more lemmas to this file at some point, because `antidiagonal`s are really meant to be summed over.

Reply:
I'm really not sure it needs to be added to default.

Reply:
Sorry, I wasn't clear. First, note that `import algebra.big_operators` imports `algebra/big_operators/default.lean` which is just a file that tries to import everything in the `algebra/big_operators` directory. I think we should add this new file to `algebra/big_operators/default.lean`, but we can't do that as it stands because this would lead to a circular import. 

Does the following suggestion work?
```suggestion
import algebra.big_operators.basic
```

Reply:
Looks like it worked! 🎉  

Please add `import algebra.big_operators.nat_antidiagonal` to `algebra/big_operators/default.lean` before merging.